tests/monotouch-test/SpriteKit/SKTransformNodeTest.cs

```diff
@@ -0,0 +1,72 @@
+﻿
+using System;
+#if XAMCORE_2_0
+using Foundation;
+#if !MONOMAC
+using UIKit;
+#endif
+using SpriteKit;
+using ObjCRuntime;
+#else
+using MonoTouch.Foundation;
+using MonoTouch.SpriteKit;
+using MonoTouch.UIKit;
+using MonoTouch.ObjCRuntime;
+#endif
+using OpenTK;
+using NUnit.Framework;
+
+namespace MonoTouchFixtures.SpriteKit {
+
+	[TestFixture]
+	[Preserve (AllMembers = true)]
+	public class SKTransformNodeTest {
+		[SetUp]
+		public void VersionCheck ()
+		{
+			if (!TestRuntime.CheckXcodeVersion (9,0))
+				Assert.Inconclusive ("Requires Xcode9+");
+		}
+
+		[Test]
+		public void EulerAngles ()
+		{
+			Vector3 V3;
+
+			using (var obj = new SKTransformNode ())
+			{
+				Asserts.AreEqual (Vector3.Zero, obj.EulerAngles, "1 EulerAngles");
+				V3 = new Vector3 (1, 2, 3);
+				obj.EulerAngles = V3;
+				Assert.AreEqual (-2.14159298f, obj.EulerAngles.X, "#x1");
```

Comment:
You're setting a vector of [1,2,3] and getting something back that looks like random numbers. Are you sure the test isn't finding a bug somewhere?